exercises/wordy/example.tt

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env ruby
```

Comment:
I don't think a conclusion was ever reached, you could try leaving it out and see if anyone notices.